pkg/controller/pxcbackup/controller.go

```diff
@@ -191,6 +191,7 @@ func (r *ReconcilePerconaXtraDBClusterBackup) Reconcile(_ context.Context, reque
 
 	var destination string
 	var s3status *api.BackupStorageS3Spec
+	var azurestatus *api.BackupStorageAzureSpec
```

Comment:
for better readability: 
```suggestion
	var azureStatus *api.BackupStorageAzureSpec
```

the variable is used in multiple places